appveyor.yml

```diff
@@ -9,28 +9,35 @@ environment:
   matrix:
 
     - TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\HastyNoise
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
     - TOOLCHAIN: "nmake-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\HastyNoise
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
     - TOOLCHAIN: "vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
+      PROJECT_DIR: examples\HastyNoise
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
-    - TOOLCHAIN: "vs-14-2015-sdk-8-1"
-      PROJECT_DIR: examples\foo
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+#https://ci.appveyor.com/project/caseymcc/hunter/builds/23954303/job/0s0cxnijdqqljc6f, alignment issues
+#    - TOOLCHAIN: "vs-14-2015-sdk-8-1"
+#      PROJECT_DIR: examples\HastyNoise
+#      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
 
-    - TOOLCHAIN: "mingw-cxx17"
-      PROJECT_DIR: examples\foo
+    - TOOLCHAIN: "vs-14-2015-win64"
```

Comment:
Please move it below, to the `# Extra {` block. E.g.:
- https://github.com/ingenue/hunter/blob/896d92591604a84231f9e2671479efe39d064fe0/.travis.yml#L95-L114

Reply:
There are no blocks in the appveyor files do you want me to create a "windows{}" one for the current, then an "extra {}" one for vs-14-2015-win64?

Reply:
> There are no blocks in the appveyor files

Because there is no need to have one in default toolchain set.

> do you want me to create a "windows{}" one for the current, then an "extra {}" one for vs-14-2015-win64?

There are `Linux`/`OSX` blocks because `.travis.yml` is used to build different OSes; `appveyor.yml` is only for Windows. Just having `Extra` is fine:
- https://github.com/ingenue/hunter/blob/9b2d862e08f674fafbaf840836559db1b9a5ddb0/appveyor.yml#L35-L41